src/adapters/destination-adapters/ChainlinkDestinationAdapter.sol

```diff
@@ -34,21 +35,37 @@ abstract contract ChainlinkDestinationAdapter is DiamondRootOval, IAggregatorV3
      * @return timestamp The timestamp of the latest answer.
      */
     function latestTimestamp() public view override returns (uint256) {
-        (, uint256 timestamp) = internalLatestData();
+        (, uint256 timestamp,) = internalLatestData();
         return timestamp;
     }
 
     /**
-     * @notice Returns an approximate form of the latest Round data. This does not implement the notion of "roundId" that
-     * the normal chainlink aggregator does and returns hardcoded values for those fields.
-     * @return roundId The roundId of the latest answer, hardcoded to 1.
+     * @notice Returns the latest Round data.
+     * @return roundId The roundId of the latest answer (sources that do not support it hardcodes to 1).
      * @return answer The latest answer in the configured number of decimals.
      * @return startedAt The timestamp when the value was updated.
      * @return updatedAt The timestamp when the value was updated.
-     * @return answeredInRound The roundId of the round in which the answer was computed, hardcoded to 1.
+     * @return answeredInRound The roundId of the round in which the answer was computed (sources that do not support it
+     * hardcodes to 1).
```

Comment:
Questioning the assumption to hardcode to 1: would 0 be better? or uint256 max? the value `1` could be a legit round number? is it easier to say that things are not supported if its 0?

Reply:
Chainlink rounds start at 1, so its safer to use, just in case an integrating contract has some validation checks based on roundId>0

Reply:
but is that not further a reason to use 0? 1 _is_ a valid chainlink round. if something does not support it we should return a value that can never be confused as a chainlink one. 0 can _never_ be a chainlink value.

Reply:
that risks client reverting even if answer and timestamp are valid. That's why we had it hardcoded to 1 in the previous implementation.

Reply:
so the goal is to never revert on inbound calls. yes, I recall that design decision. the tradeoff here is sometimes we might return data that is partially incomplete/incorrect (a roundId that is "valid" of 1, but does not actually mean it was the first round; it was the hardcoded default) but we never revert.

Reply:
One more point here. This is what Redstone and other oracles do.